petfinder-data/config.py

```diff
@@ -7,3 +7,4 @@
 PETFINDER_SECRET = os.getenv('PETFINDER_SECRET')
 DATABASE_URL = os.getenv('DATABASE_URL')
 HEROKU_URL = os.getenv('HEROKU_POSTGRESQL_AMBER_URL')
+City_State = os.getenv("LOCATION")
```

Comment:
Do you envision the city/state variable as being generally static (not frequently changed), or something to change? If it's something that people may want to be able to easily change, one option could be to omit it as an environment variable, and instead add it as an argument when we call the `data_runner` / `data_getter` scripts.

For example, instead of just calling `python data_getter.py` we could call `python data_getter.py --location Chicago, IL`.
[Here's](https://click.palletsprojects.com/en/8.1.x/) one package I've used for this.

Reply:
Removed City_State from env and made it a CLI.